From fbd70083ebdee7e65879f68e35331992cf7d1436 Mon Sep 17 00:00:00 2001
From: Matthias Proske <Matthias.Proske@bshg.com>
Date: Wed, 26 Jun 2019 18:24:24 +0200
Subject: [PATCH] Base Password: create group IDs for keys and certs

GID 400 and 401 are allocated for BSH-new groups.
This is created statically and will be used on all images that make use
of the meta-bsh-core layer. (It will end up in ELP, Recovery, EMS)
---
 group.master | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/group.master b/group.master
index cea9d60..a280bb6 100644
--- a/group.master
+++ b/group.master
@@ -38,4 +38,6 @@ staff:*:50:
 games:*:60:
 shutdown:*:70:
 users:*:100:
+bsh-keys:*:400:
+bsh-certs:*:401:
 nogroup:*:65534:
-- 
2.21.0

